.env.example

```diff
@@ -0,0 +1,5 @@
+# Example environment variables for local development
+
+# HTTP listen address for the API (matches internal/config)
+# If unset, the service defaults to :8080
+HTTP_ADDR=:8080
```

Comment:
How would you use secret environment variables in a production-grade application?